core/src/main/java/tc/oc/pgm/modules/ModifyBowProjectileModule.java

```diff
@@ -59,11 +69,16 @@ public ModifyBowProjectileModule parse(MapFactory factory, Logger logger, Docume
           potionEffects.add(XMLUtils.parsePotionEffect(potionElement));
           changed = true;
         }
+
+        for (Element noPickupElement : parent.getChildren("pickup")) {
+          pickupFilter = filters.parseFilterProperty(noPickupElement, "filter");
+          changed = true;
+        }
```

Comment:
This parses multiple children, taking just the last one. This is incorrect behavior, either ensure there's a single child or combine the filters (i'd just do the former, it's simpler)